src/Native/Unix/System.Security.Cryptography.Native/CMakeLists.txt

```diff
@@ -82,13 +85,20 @@ set_target_properties(System.Security.Cryptography.Native.OpenSsl-Static PROPERT
 set_target_properties(System.Security.Cryptography.Native.OpenSsl-Static PROPERTIES OUTPUT_NAME System.Security.Cryptography.Native.OpenSsl CLEAN_DIRECT_OUTPUT 1)
 
 if (FEATURE_DISTRO_AGNOSTIC_SSL)
-    add_custom_command(TARGET System.Security.Cryptography.Native.OpenSsl POST_BUILD
-        COMMENT "Verifying System.Security.Cryptography.Native.OpenSsl.so dependencies"
-        COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/../verify-so.sh 
-            $<TARGET_FILE:System.Security.Cryptography.Native.OpenSsl> 
-            "Verification failed. System.Security.Cryptography.Native.OpenSsl.so has undefined dependencies. These are likely OpenSSL APIs that need to be added to opensslshim.h"
-        VERBATIM
-    )
+    # on macOS the link step fails with undefined symbols, and the script doesn't run.
```

Comment:
>  # on macOS the link step fails with undefined symbols, and the script doesn't run.

I read this as "the link step is going to fail, reporting undefined symbols, and the script won't run" but I think I should have read this as, "if there's undefined symbols causing the link step to fail, the script won't run" .. which might be clearer? Similar for LInux.

Reply:
"the link step is going to fail, reporting undefined symbols, and the script won't run" is correct.